src/Razor/src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/VisualStudioMacLSPEditorFeatureDetector.cs

```diff
@@ -50,8 +50,13 @@ internal VisualStudioMacLSPEditorFeatureDetector()
         {
         }
 
-        public override bool IsLSPEditorAvailable(string documentFilePath, object? hierarchy)
+        public override bool IsLSPEditorAvailable(string? documentFilePath, object? hierarchy)
```

Comment:
@davidwengier  Maybe rename to `documentMoniker` to sync name with the base method? Or this is different in VSMac and current name is better here?

Reply:
I've no idea.. certainly "moniker" is a VS-ism as far as I'm concerned. If the build fails, I'll update it, but I'm not fussed.